keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -414,48 +418,73 @@ Both these cases will be handled properly, here's an example:
 type Object struct {
      // This field has an invalid zero-value, empty string is not a
      // valid string.
-     // +default=default-name
-     Name string
+     //
+     // If this is a CRD, there would be a kubebuilder specific tag, and
+     // if this is a built-in, there would be a specific validation function,
+     // but basically:
+     // +minLength=1
```

Comment:
what I don't like about these constraints is that kubectl will follow them and a native PodSpec will be accepted while a CRD embedded PodSpec will be rejected.